src/plugins/label/label-plugin.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button } from "@components/Button";
+import { AnoriPlugin, WidgetConfigurationScreenProps, WidgetRenderProps, WidgetDescriptor } from "@utils/user-data/types";
+import { Input } from "@components/Input";
+import { Textarea } from "@components/Input";
+import './styles.scss';
+import { useTranslation } from "react-i18next";
+import { useRef, useState } from "react";
+import { WidgetExpandAreaRef } from "@components/WidgetExpandArea";
+import { translate } from "@translations/index";
+
+type labelWidgetConfigType = {
+    title: string,
+    description: string,
+};
+
+const LabelScreen = ({ config, instanceId }: WidgetRenderProps<labelWidgetConfigType>) => {
+    const [showExpandableArea, setShowExpandableArea] = useState(false);
+    const expandAreaRef = useRef<WidgetExpandAreaRef>(null);
+
+    return (<div className="ExpandableTestWidget" onClick={() => showExpandableArea ? expandAreaRef.current?.focus(true) : setShowExpandableArea(true)}>
+        <div className="text">
+            <h1 className="label">{ config.title }</h1>
+            <div className="label_span">{ config.description }</div>
+        </div>
+    </div>);
+};
+
+const labelConfigurationScreen = ({ saveConfiguration, currentConfig }: WidgetConfigurationScreenProps<labelWidgetConfigType>) => {
```

Comment:
Could you update this to be `LabelConfigurationScreen` as this is React component